src/Form/_index.scss

```
@@ -46,6 +46,18 @@ $select-icon-padding: .5625rem !default;
   flex-grow: 1;
   align-items: flex-start;
 
+  .form-control {
+    min-height: $input-height;
+  }
+
+  .form-control-lg {
+    min-height: $input-height-lg;
+  }
+
+  .form-control-sm {
+    min-height: $input-height-sm;
+  }
+
```

It seems to me that the option of specifying a minimum height only for the `textarea` would be better because in your implementation we have a `min-height` for all `.form-controls` that do not need it.

```suggestion
  textarea {
    &.form-control {
      min-height: $input-height;
    }
    
    &.form-control-lg {
      min-height: $input-height-lg;
    }
    
    &.form-control-sm {
      min-height: $input-height-sm;
    }
  }
```

---

I agree, other types of inputs cannot have problems with height.